environments/icds-cas/public.yml

```diff
@@ -166,6 +166,29 @@ additional_syslog_config:
 
 kafka_log_retention: 504 # 21 days
 
+# Promethues,Alertmanager,grafana,exporter settings
+prometheus_db_dir: "{{ encrypted_root }}/prometheus"
+prometheus_storage_retention: "365d"
+prod_mode: True
```

Comment:
what does this do?

Reply:
https://github.com/dimagi/commcare-prometheus/blob/master/roles/prometheus/tasks/main.yml#L37 probably need to change var name for something more meaningful. It's there because it's not possible to modify host file in docker.